Frameworks/include/UIApplicationInternal.h

```diff
@@ -16,6 +16,9 @@
 #pragma once
 
 #import <UIKit/UIApplication.h>
+#import <UWP/WindowsFoundation.h>
```

Comment:
I prefer doing the right thing and fixing issues preventing us from doing the right thing rather than leaving scary things hidden for someone else to deal with. If there was "something funky" lets address that rather than running from it. Additionally this is an Internal header and thus shouldn't be subjected to modules. 

---
In reply to: [82297724](https://github.com/Microsoft/WinObjC/pull/1100#discussion_r82297724) [](ancestors = 82297724)